dev-python/semantic_version/semantic_version-2.8.5-r1.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
```

Comment:
You've lost the django dep here. But then, this makes it possible to merge it as-is for the time being, and add django when it's 3.10-ready. I'll file a bug not to forget about it.

Reply:
Could you link the bug here?

Reply:
Sure. https://bugs.gentoo.org/791367